StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSheet/CustomerSheet.swift

```diff
@@ -86,6 +86,7 @@ public class CustomerSheet {
     public func present(from presentingViewController: UIViewController,
                         completion csCompletion: @escaping (CustomerSheetResult) -> Void
     ) {
+        AnalyticsHelper.shared.generateSessionID()
```

Comment:
Paymentsheet, generateSesionID - generateSessionID() happens in init
PaymehtSheet.FlowController - generateSessionID() happens in create()
CustomerSheet - generateSessionID() happens in present()

For consistency, seems like `generateSessionID()` should all be in `init` or `present` across the board?

Reply:
That's fair, updated.